src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -81,6 +81,11 @@
           text: az storage container create -n MyStorageContainer --fail-on-exist
 """
 
+helps['storage account create'] = """
```

Comment:
Did this not have help text before or is this some kind of side-effect of this new approach (like we can't get docstrings via reflection or something like that)?